src/components/battles/animations/shared/Moves.ts

```diff
@@ -37,7 +37,14 @@ export class Moves<TGameStartr extends FullScreenPokemon> extends GeneralCompone
      */
     public playMove(teamAndAction: ITeamAndAction<IMoveAction>, callback: () => void): void {
         const battleInfo: IBattleInfo = this.gameStarter.battleMover.getBattleInfo() as IBattleInfo;
-        const animatorType: typeof Move = this.movesBag[teamAndAction.action.move] || this.movesBag.default;
+        let animatorType: typeof Move = this.movesBag.default;
+        if (teamAndAction.action.move === "Scratch") {
+            animatorType = this.movesBag.SCRATCH;
+        } else if (teamAndAction.action.move === "Slash") {
```

Comment:
This extra logic should be unnecessary given your changes to `MovesBag.ts`. We shouldn't hardcode for moves here.